app/Console/Commands/AuditDependencies.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class AuditDependencies extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'dependencies:audit';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Run security audit on CDash\'s dependencies';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Handle the running of the command.
+     *
+     * Echo a header and then the output of each command.
+     * Laravel will pipe the output to a file if run by the scheduler
+     * or it will print to the console if run manually.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        //  PHP auditing via composer
+        $output=null;
+        echo("\n\nComposer Report:\n\n");
+        exec("HOME=" . ENV("HOME") . " composer audit --no-interaction", $output);
+        print(implode("\n", $output));
```

Comment:
`echo()` is used on line 46, while `print()` is used on line 48.  They both effectively do the same thing.  Is there a reason for this inconsistency?